content/en/storage-provider/skills/storage/index.md

```diff
@@ -12,20 +12,50 @@ menu:
 weight: 640
 toc: true
 ---
-Storage skills might seem the most obvious skill requirement to run a Storage Provider. In contrast to classical storage arrays and classical storage administration however, Filecoin requires an end-to-end skillset for running a 24/7 application.
+Storage is a critical component of running a successful Storage Provider in the Filecoin network. While it may seem obvious that having strong storage skills is important, Filecoin requires a unique end-to-end skill set to run a 24/7 application.
```

Comment:
```suggestion

Storage is a critical component of running a successful storage provider in the Filecoin network. While it may seem obvious that having strong storage skills is important, Filecoin requires a unique end-to-end skill set to run a 24/7 application.
```

Keep an empty line after the front-matter. The term _storage provider_ should not be capitalized.